app/Admin/Settings.php

```diff
@@ -28,6 +28,22 @@ class Settings {
      */
 	protected static $theme_options;
 
+	public function hooks() {
+		add_action( 'carbon_fields_register_fields', [$this, 'crb_attach_theme_options'], 0, 0 );
+		add_filter( 'diviner_js_config', [ $this, 'custom_diviner_js_config' ] );
+	}
+
+	public function custom_diviner_js_config( $data  ) {
+
+		$settings = [
+			'permission_notice' => carbon_get_theme_option(self::FIELD_GENERAL_PERMISSIONS),
```

Comment:
use `static::<constant>` here, and wherever you use a constant, in place of `self::<constant>` to allow for easier extension in the future.
Furthermore: is there a fallback default here, or in the JS code, if the option is empty?

Reply:
Done